velox/docs/functions/presto/string.rst

```diff
@@ -255,6 +255,14 @@ String Functions
 
     Converts ``string`` to uppercase.
 
+.. function:: word_stem(word) -> varchar
+
+    Returns the stem of ``word`` in the English language.
```

Comment:
What happens if 'word' is not an English word?

Reply:
it returns the same value

Reply:
Please, update the documentation to clarify.

Reply:
fixed in the new commit